xscreensaver: update to version 6.10

klausz65 opened the pull request.

klausz65 committed:
xscreensaver: update to version 6.10

klausz65 commented:
Hmm, I have no idea why jenkins wants to rebuild in patches subdir:
jenkinshelper: done with components/desktop/xscreensaver return code 0
+ [ rc -ne 0 ]
+ echo 'jenkinshelper: building for components/desktop/xscreensaver/patches...'
jenkinshelper: building for components/desktop/xscreensaver/patches...
+ pwd
+ curpwd=/ws/jenkins/workspace/rganisation_oi-userland_PR-24603
+ cd components/desktop/xscreensaver/patches
+ gmake clean
gmake: *** No rule to make target 'clean'.  Stop.
+ rc=2
+ cd /ws/jenkins/workspace/rganisation_oi-userland_PR-24603
+ echo 'jenkinshelper: done with components/desktop/xscreensaver/patches return code 2'
jenkinshelper: done with components/desktop/xscreensaver/patches return code 2
+ [ rc -ne 0 ]
+ worst=2
+ return 2
+ rt=2
+ exit 2
script returned exit code 2

AndWac commented:
There is another problem when I build it here:
make[1]: Leaving directory '/export/home/andreas/oi-userland/components/desktop/xscreensaver/build/amd64/po'
( mkdir -p /export/home/andreas/oi-userland/components/desktop/xscreensaver/build/prototype/i386/usr/share/X11/app-defaults ; mv /export/home/andreas/oi-userland/components/desktop/xscreensaver/build/prototype/i386/usr/X11R6/lib/X11/app-defaults/XScreenSaver /export/home/andreas/oi-userland/components/desktop/xscreensaver/build/prototype/i386/usr/share/X11/app-defaults)
mv: cannot stat '/export/home/andreas/oi-userland/components/desktop/xscreensaver/build/prototype/i386/usr/X11R6/lib/X11/app-defaults/XScreenSaver': No such file or directory
gmake: *** [/export/home/andreas/oi-userland/make-rules/common-rules.mk:68: /export/home/andreas/oi-userland/components/desktop/xscreensaver/build/amd64/.installed] Error 1

klausz65 commented:
The /usr/X11R6/lib/X11/app-defaults/XScreenSaver': No such file or directory issue is fixed, somehow our build-zone can't detect the app-defaults directory correctly - I removed the previous POST_INSTALL action and replaced it with the configure option for the app-defaults directory - on two other systems one x64 and one SPARC SunFire V240 the package has been build correctly - please re-tray again - thanks.
